packages/resolve-bus-memory/src/index.js

```diff
@@ -0,0 +1,34 @@
+const eventHandlersMap = new Map();
```

Comment:
Why singleton?

It is a responsibility of the bus user - to pass same instance of bus to each place where it is required.